akka-stream-tests/src/test/scala/akka/stream/scaladsl/FromMaterializationSpec.scala

```diff
@@ -46,7 +46,7 @@ class FromMaterializerSpec extends StreamSpec {
         }
         .named("my-name")
 
-      source.runWith(Sink.head).futureValue shouldBe Some("my-name")
+      source.runWith(Sink.head).futureValue shouldBe Some("setup-my-name")
```

Comment:
The name attribute is special and is concatenated, so since I added a default name "setup" it now becomes "setup-my-name" when adding "my-name"